skins/foundation/js/2.cubecart.js

```diff
@@ -342,6 +343,13 @@ function price_inc_options() {
                 var prices = $.parseJSON(returned.responseText);
                 $('#ptp').html(prices[0]);
                 $('#fbp').html(prices[1]);
+                if (absolute && prices[0] <= prices[1]) {
```

Comment:
The values in the prices array are strings beginning with a monetary symbol such as '$'; we can parse out the actual numeric values to compare, but I didn't at this time as the comparison appears to function as desired with the current implementation.